code/modules/materials/recipes_items.dm

```diff
@@ -287,3 +287,15 @@
 /datum/stack_recipe/gauntlets
 	title = "gauntlets"
 	result_type = /obj/item/clothing/gloves/thick/craftable
+
+/datum/stack_recipe/paper_sheets
+	title          = "sheet of paper"
+	result_type    = /obj/item/paper
```

Comment:
In the paperwork PR the place result proc is actually overriden to add the sheets to an existing paper bundle. But if I add the paper bundle changes I made in here, it means I'd have to include a lot of the paperwork PR.. So yeah, this is only temporary until we can merge my paperwork stuff, since you guys wanted me to cut it into smaller parts.